README.md

```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
 ```javascript
 npm install ng2-eonasdan-datetimepicker --save
 ```
-
-2) Add the A2Edatetimepicker module import in your module.:
+2) Include all required scripts if it doesn't included yet in your project:
```

Comment:
There are some typos in the text general and it lacks styling a bit.

I may help you here a bit, when you confirm these configurations work in both scenarios.
Or when https://github.com/atais/ng2-eonasdan-datetimepicker/issues/29 is resolved

Reply:
You can edit it :smile: but I can confirm  #29 will be resolved, I tested with webpack & angular 5 with these configurations.

Reply:
OK I will fix it and merge this changes.